site/styles/_pages.scss

```diff
@@ -15,6 +15,9 @@
 .page-header {
   padding-right: 20px;
   padding-left: 20px;
+  @media (min-width: 438px) {
```

Comment:
438px is a bit of an arbitrary breakpoint which doesn't match any of the widths specified in the variables file and I wonder what problem this tries to solve.